main.tf

```diff
@@ -260,3 +260,40 @@ resource "aws_security_group_rule" "cidr_ingress" {
   cidr_blocks       = var.allowed_cidr_blocks
   security_group_id = local.rds_security_group_id
 }
+
+################################################################################
+# Cluster Endpoints
+################################################################################
+
+resource "aws_rds_cluster_endpoint" "custom_reader" {
+  count = var.create_cluster && var.create_cluster_custom_endpoints ? 1 : 0
```

Comment:
this looks interesting, thank you for the PR! I haven't had much time to look in detail or test but just at first pass - this looks like we should `for_each` over a map of endpoint configs, that would be a more scalable config for any number of endpoints. what do you think?

Reply:
I thought about it
Give me some time to test this configuration
Now I do not think that it is working :)

Reply:
Yes, probably iterating through for_each will be more flexible
and at the same time will support the WRITE type, which can only be used in a multi-master cluster